docs/en/setup/service-agent/java-agent/README.md

```diff
@@ -153,8 +153,8 @@ property key | Description | Default |
 `plugin.dubbo.provider_arguments_length_threshold`| When `plugin.dubbo.collect_provider_arguments` is `true`, Arguments of length from the front will to the OAP backend |`256`| 
 `plugin.kafka.bootstrap_servers`| A list of host/port pairs to use for establishing the initial connection to the Kafka cluster. | `localhost:9092`|
 `plugin.kafka.get_topic_timeout`| Timeout period of reading topics from the Kafka server, the unit is second. |`10`|
-`plugin.kafka.consumer_config`| Kafka producer configuration. ||
```

Comment:
I think this still works, right?

Reply:
Not, just forget to remove it from docs. This was originally reserved for `profile task` communication with OAP. But we discard this way. Agent communicate with OAP one way.

Reply:
Not, just forget to remove it from docs. This was originally reserved for `profile task` communication with OAP. But we discard this way. Agent communicate with OAP one way.